packages/component-library/src/components/layout/mt-empty-state/mt-empty-state.vue

```diff
@@ -51,9 +52,11 @@ withDefaults(
     linkText?: string;
     linkType?: "external" | "internal";
     buttonText?: string;
+    leftAligned?: boolean;
```

Comment:
Also, add this deprecated tag here as well. That way devs see it right away in their IDE.